DependencyInjection/Configuration.php

```diff
@@ -30,7 +27,7 @@ public function getConfigTreeBuilder()
         $rootNode
             ->children()
                 ->booleanNode('autoinclude')->defaultTrue()->end()
-                ->scalarNode('base_path')->defaultValue('bundles/norzechowiczaceeditor/ace')->end()
+                ->scalarNode('base_path')->defaultValue('vendor/ace')->end()
```

Comment:
also why in readme there is a not that this bundle expects `web/ace` and default base_path is `vendor/ace` ?